src/lib/components/panel.svelte

```diff
@@ -8,10 +8,10 @@
 
 <style lang="postcss">
   .panel {
-    @apply rounded-xl border-2  p-8;
+    @apply rounded-xl border-2 p-8 dark:border-subtle;
   }
 
   .error {
-    @apply border-red-700 bg-red-50;
+    @apply border-error bg-error;
   }
```

Comment:
Looks like `border-error` here ⬇️  is being overridden by `border-subtle` in dark mode

<img width="457" alt="Screenshot 2024-04-09 at 8 28 19 PM" src="https://github.com/temporalio/ui/assets/15069288/e8019b37-6273-487c-9a53-9110f9f01019">


Moving the `error` class should fix it

```js
  .panel {
    @apply rounded-xl border-2 p-8 dark:border-subtle;
    &.error {
      @apply border-error bg-error;
    }
  }
```

Reply:
Thanks Laura! I appreciate you!